db/migrate/20160303115326_add_websocket_to_miq_server.rb

```diff
@@ -0,0 +1,5 @@
+class AddWebsocketToMiqServer < ActiveRecord::Migration[5.0]
+  def change
+    add_column :miq_servers, :has_active_websocket, :boolean
```

Comment:
There are a few columns on Host and MiqServer (I think, to support the existing console stuff...can those be removed?...not necessarily for this PR)